### PR TITLE
Group error rows for bulk results uploader

### PR DESCRIPTION
**Pull request by zdeveloper**
# FRONTEND PULL REQUEST

## Related Issue

- #5416

## Changes Proposed

- Detailed explanation of what this PR should do

## Additional Information

- decisions that were made
- notice of future work that needs to be done

## Testing

- How should reviewers verify this PR?

## Screenshots / Demos

- For large changes, please pair with a designer to ensure changes are as intended

### CODE REVIEW

File: frontend/src/app/testResults/uploads/Uploads.tsx [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import React, { ReactElement, useEffect, useState } from "react";
 import { useSelector } from "react-redux";
 import { Button, FormGroup } from "@trussworks/react-uswds";
+import { Maybe } from "graphql/jsutils/Maybe";
```
Comment (nathancrtr) [inline frontend/src/app/testResults/uploads/Uploads.tsx:4]: hell yeah 👨‍🎤

**Review by BobanL — COMMENTED** [review]
Mostly looks good, but for general errors without rows the line doesn't cover the whole table
image

**Review by BobanL — APPROVED** [review]
Looks good to me!

**Review by DanielSass — APPROVED** [review]
looks great

**Pull request merged**